src/__tests__/lib/containers/LoadingScreen.test.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react'
+import { BlockingLoader, BlockingLoaderProps } from '../../../lib/containers/LoadingScreen'
+import { mount } from 'enzyme'
```

Comment:
Could you use `@testing-library/react` ? It forces us to consider accessibility, and Enzyme isn't well supported anymore (doesn't have full support for React 16 and no support yet for React 17)